### ISSUE TITLE
New tflint don't have --loglevel CLI flag

### ISSUE THREAD

**Issue by nantiferov** [github/super-linter#3406]
## Describe the bug

tflint was updated in v4.9.6 and `loglevel` option was removed in that version. Now it adds this warning:

```log
[WARN]   ERROR! Failed to run:[tflint --init] at location:[/github/workspace]
[75](https://github.com/bolteu/terraform-cognito/actions/runs/3173283267/jobs/5168720139#step:9:76)
[WARN]   BUILD_CMD:[Failed to parse CLI options; `loglevel` option was removed in v0.40.0. Please set `TFLINT_LOG` environment variables instead]
```
https://github.com/terraform-linters/tflint/releases/tag/v0.40.0

## Expected behavior

tflint don't produce warnings in log regarding this missing option.

## Steps to Reproduce

Add super-linter v4.9.6 to workflow like in default example and get warning message about missing cli option.

## Logs

```log
[WARN]   ERROR! Failed to run:[tflint --init] at location:[/github/workspace]
[75](https://github.com/bolteu/terraform-cognito/actions/runs/3173283267/jobs/5168720139#step:9:76)
[WARN]   BUILD_CMD:[Failed to parse CLI options; `loglevel` option was removed in v0.40.0. Please set `TFLINT_LOG` environment variables instead]
```

## Additional context

P.S. Seems like something is off with changelog. v4.9.6 has tflint 0.40.x, while release notes says `Bump terraform-linters/tflint-bundle from v0.38.1.0 to v0.39.1.1`
https://github.com/github/super-linter/releases/tag/v4.9.6
This PR is missing from changelog https://github.com/github/super-linter/pull/3339

**Comment by ferrarimarco** [comment]
#3414 should take care of this issue. :)

**Comment by ferrarimarco** [comment]
And #3415 should help us catching this kind of errors in the future.

**Comment by mcaulifn** [comment]
Is there an estimated time to release this fix? I see it was fixed just after the last release.

**Comment by ferrarimarco** [comment]
@lindluni might be the best person to answer this. Closing because the PR was merged.

**Comment by yermulnik** [comment]
I can still see this in GH action output when using `github/super-linter/slim`. Is this expected?
```
2022-12-02 20:20:53 [WARN]   BUILD_CMD:[Failed to parse CLI options; `loglevel` option was removed in v0.40.0. Please set `TFLINT_LOG` environment variables instead
```
Config:
```yml
# https://github.com/github/super-linter
name: Lint Code Base

on: # yamllint disable-line rule:truthy
  push:
    branches:
      - master
  pull_request:
    branches:
      - master

jobs:
  build:
    name: Lint Code Base
    runs-on: ubuntu-latest
    steps:
      - name: Checkout Code
        uses: actions/checkout@v3
        with:
          # Full git history is needed to get a proper
          # list of changed files within `super-linter`
          fetch-depth: 0
      - name: Lint Code Base
        uses: github/super-linter/slim@v4
        env:
          DEFAULT_BRANCH: master
          DISABLE_ERRORS: false
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          TF_TOKEN_app_terraform_io: ${{ secrets.TF_TOKEN_APP_TERRAFORM_IO }}
          VALIDATE_ALL_CODEBASE: false
          VALIDATE_BASH: true
          VALIDATE_BASH_EXEC: true
          VALIDATE_CLOUDFORMATION: true
          VALIDATE_DOCKERFILE_HADOLINT: true
          VALIDATE_GITHUB_ACTIONS: true
          VALIDATE_GITLEAKS: true
          VALIDATE_GO: true
          VALIDATE_GROOVY: true
          VALIDATE_HTML: true
          VALIDATE_JSON: true
          VALIDATE_MARKDOWN: true
          VALIDATE_PERL: true
          VALIDATE_PHP: true
          VALIDATE_POWERSHELL: true
          VALIDATE_PYTHON: true
          VALIDATE_PYTHON_BLACK: true
          VALIDATE_PYTHON_FLAKE8: true
          VALIDATE_PYTHON_ISORT: true
          VALIDATE_PYTHON_MYPY: true
          VALIDATE_PYTHON_PYLINT: true
          VALIDATE_RUBY: true
          VALIDATE_SQL: true
          VALIDATE_SQLFLUFF: true
          VALIDATE_TERRAFORM_TERRASCAN: true
          VALIDATE_TERRAFORM_TFLINT: true
          VALIDATE_TERRAGRUNT: true
          VALIDATE_XML: true
          VALIDATE_YAML: true
```
Related excerpts from GH action output:
```shell
  /usr/bin/docker pull ghcr.io/github/super-linter:slim-v4.9.7
[…]
  Digest: sha256:900277f36d47d5ddc460d901ea9dfcb1d348f7390066f800a0895cd88866b31f
  Status: Downloaded newer image for ghcr.io/github/super-linter:slim-v4.9.7
  ghcr.io/github/super-linter:slim-v4.9.7
[…]
Run github/super-linter/slim@v4
/usr/bin/docker run --name ghcriogithubsuperlinterslimv497_eaa581 --label 290506 --workdir /github/workspace --rm -e "DEFAULT_BRANCH" -e "DISABLE_ERRORS" -e "GITHUB_TOKEN" -e "TF_TOKEN_app_terraform_io" -e "VALIDATE_ALL_CODEBASE" -e "VALIDATE_BASH" -e "VALIDATE_BASH_EXEC" -e "VALIDATE_CLOUDFORMATION" -e "VALIDATE_DOCKERFILE_HADOLINT" -e "VALIDATE_GITHUB_ACTIONS" -e "VALIDATE_GITLEAKS" -e "VALIDATE_GO" -e "VALIDATE_GROOVY" -e "VALIDATE_HTML" -e "VALIDATE_JSON" -e "VALIDATE_MARKDOWN" -e "VALIDATE_PERL" -e "VALIDATE_PHP" -e "VALIDATE_POWERSHELL" -e "VALIDATE_PYTHON" -e "VALIDATE_PYTHON_BLACK" -e "VALIDATE_PYTHON_FLAKE8" -e "VALIDATE_PYTHON_ISORT" -e "VALIDATE_PYTHON_MYPY" -e "VALIDATE_PYTHON_PYLINT" -e "VALIDATE_RUBY" -e "VALIDATE_SQL" -e "VALIDATE_SQLFLUFF" -e "VALIDATE_TERRAFORM_TERRASCAN" -e "VALIDATE_TERRAFORM_TFLINT" -e "VALIDATE_TERRAGRUNT" -e "VALIDATE_XML" -e "VALIDATE_YAML" -e "HOME" -e "GITHUB_JOB" -e "GITHUB_REF" -e "GITHUB_SHA" -e "GITHUB_REPOSITORY" -e "GITHUB_REPOSITORY_OWNER" -e "GITHUB_RUN_ID" -e "GITHUB_RUN_NUMBER" -e "GITHUB_RETENTION_DAYS" -e "GITHUB_RUN_ATTEMPT" -e "GITHUB_ACTOR" -e "GITHUB_TRIGGERING_ACTOR" -e "GITHUB_WORKFLOW" -e "GITHUB_HEAD_REF" -e "GITHUB_BASE_REF" -e "GITHUB_EVENT_NAME" -e "GITHUB_SERVER_URL" -e "GITHUB_API_URL" -e "GITHUB_GRAPHQL_URL" -e "GITHUB_REF_NAME" -e "GITHUB_REF_PROTECTED" -e "GITHUB_REF_TYPE" -e "GITHUB_WORKSPACE" -e "GITHUB_ACTION" -e "GITHUB_EVENT_PATH" -e "GITHUB_ACTION_REPOSITORY" -e "GITHUB_ACTION_REF" -e "GITHUB_PATH" -e "GITHUB_ENV" -e "GITHUB_STEP_SUMMARY" -e "GITHUB_STATE" -e "GITHUB_OUTPUT" -e "RUNNER_OS" -e "RUNNER_ARCH" -e "RUNNER_NAME" -e "RUNNER_TOOL_CACHE" -e "RUNNER_TEMP" -e "RUNNER_WORKSPACE" -e "ACTIONS_RUNTIME_URL" -e "ACTIONS_RUNTIME_TOKEN" -e "ACTIONS_CACHE_URL" -e GITHUB_ACTIONS=true -e CI=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/_temp/_runner_file_commands":"/github/file_commands" -v "/home/runner/work/devops-core/devops-core":"/github/workspace" ghcr.io/github/super-linter:slim-v4.9.7
[…]
2022-12-02 20:20:53 [WARN]   ERROR! Failed to run:[tflint --init] at location:[]
2022-12-02 20:20:53 [WARN]   BUILD_CMD:[Failed to parse CLI options; `loglevel` option was removed in v0.40.0. Please set `TFLINT_LOG` environment variables instead
[…]
```
